Add go-mpd

unki2aut opened the pull request:
Please check if what you want to add to `awesome-go` list meets [quality standards](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard) before sending pull request. Thanks!

**Please provide package links to:**

- github.com repo: https://github.com/unki2aut/go-mpd
- godoc.org: https://godoc.org/github.com/unki2aut/go-mpd
- goreportcard.com: https://goreportcard.com/report/github.com/unki2aut/go-mpd
- coverage service link [gocover](https://gocover.io/github.com/unki2aut/go-mpd/)


Very good coverage

**Note**: that new categories can be added only when there are 3 packages or more.

**Make sure that you've checked the boxes below before you submit PR:**
- [x] I have added my package in alphabetical order.
- [x] I have an appropriate description with correct grammar.
- [x] I know that this package was not listed before.
- [x] I have added godoc link to the repo and to my pull request.
- [x] I have added coverage service link to the repo and to my pull request.
- [x] I have added goreportcard link to the repo and to my pull request.
- [x] I have read [Contribution guidelines](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#contribution-guidelines), [maintainers note](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#maintainers) and [Quality standard](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard).

Thanks for your PR, you're awesome! :+1:

avelino commented:
Thank you for contributing with [awesome-go](https://github.com/avelino/awesome-go), we will revise your contribution as soon as possible.

Automation body content check:
- godoc.org: True
- goreportcard.com: False
- coverage: True

unki2aut commented:
@panjf2000 added features:
- Use of [unki2aut/go-xsd-types](https://github.com/unki2aut/go-xsd-types) to correctly parse `DateTime` and `Duration` values
- Added several properties to `Representation`, `AdaptationSet` and `SegmentTemplate` (e.g. `BaseURL`)
- Improved the unit tests
- Added documentation

Also created a fork as the original repo is not very active. My PR to improve `SegmentTemplate` handling (https://github.com/mc2soft/mpd/pull/2) is still open after 1.5 years.
